Patches/Core/ThingDefs_Misc/Apparel_Various.xml

```diff
@@ -38,56 +34,55 @@
 	<!-- ========== Armor vest ========== -->
 
 	<Operation Class="PatchOperationAdd">
-		<xpath>*/ThingDef[defName="Apparel_VestPlate"]/statBases</xpath>
+		<xpath>*/ThingDef[defName="Apparel_FlakVest"]/statBases</xpath>
 		<value>
 			<Bulk>5</Bulk>
 			<WornBulk>1</WornBulk>
     </value>
   </Operation>
 
 	<Operation Class="PatchOperationReplace">
-		<xpath>*/ThingDef[defName="Apparel_VestPlate"]/statBases/MaxHitPoints</xpath>
+		<xpath>*/ThingDef[defName="Apparel_FlakVest"]/statBases/MaxHitPoints</xpath>
 		<value>
 			<MaxHitPoints>125</MaxHitPoints>
     </value>
   </Operation>
 
 	<Operation Class="PatchOperationReplace">
-		<xpath>*/ThingDef[defName="Apparel_VestPlate"]/statBases/Mass</xpath>
+		<xpath>*/ThingDef[defName="Apparel_FlakVest"]/statBases/Mass</xpath>
 		<value>
 			<Mass>13</Mass>
     </value>
   </Operation>
 
 	<Operation Class="PatchOperationReplace">
-		<xpath>*/ThingDef[defName="Apparel_VestPlate"]/statBases/ArmorRating_Sharp</xpath>
+		<xpath>*/ThingDef[defName="Apparel_FlakVest"]/statBases/ArmorRating_Sharp</xpath>
 		<value>
 			<ArmorRating_Sharp>0.6</ArmorRating_Sharp>
     </value>
   </Operation>
 
   <Operation Class="PatchOperationRemove">
-  	<xpath>*/ThingDef[defName="Apparel_VestPlate"]/equippedStatOffsets/MoveSpeed</xpath>
+  	<xpath>*/ThingDef[defName="Apparel_FlakVest"]/equippedStatOffsets/MoveSpeed</xpath>
   </Operation>
 
   <Operation Class="PatchOperationAdd">
-  	<xpath>*/ThingDef[defName="Apparel_VestPlate"]/apparel/bodyPartGroups</xpath>
+  	<xpath>*/ThingDef[defName="Apparel_FlakVest"]/apparel/bodyPartGroups</xpath>
   	<value>
   		<li>Shoulders</li>
   	</value>
   </Operation>
 
   <!-- Change recipe to stuffed -->
-
-	<Operation Class="PatchOperationAdd">
-		<xpath>*/ThingDef[defName="Apparel_VestPlate"]</xpath>
+  <Operation Class="PatchOperationAdd">
+		<xpath>*/ThingDef[defName="Apparel_FlakVest"]</xpath>
 		<value>
-			<costStuffCount>90</costStuffCount>
+			<costStuffCount>60</costStuffCount>
```

Comment:
Why the balance change?

Reply:
It matches the new vanilla flak vest steel cost

Reply:
As above, we're not following vanilla changes